docs/admin-ports.md

```diff
@@ -38,6 +38,8 @@ The following is a list of ports used by internal DC/OS services, and their corr
  - 8181: dcos-exhibitor
  - 9990: dcos-cosmos
  - 15055: dcos-history-service
+ - 15101: dcos-marathon libprocess
+ - 15201: dcos-metronome libprocess
```

Comment:
No particular reason for that. Keeping the gaps in case there are more services that feet that range better. I can change them to numbers you feel are better suitable.